### PR TITLE
solution

### PR DESCRIPTION
**Pull request by maksym-kostetskyi**

### CODE REVIEW

**Review by loralevitska — CHANGES_REQUESTED** [review]
Almost done) Let's improve

File: src/convertToObject.js [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
-  // write your code here
+  const result = {};
+  const sourceStringTrimmed = sourceString.trim();
+  const stringSplit = sourceStringTrimmed.split(';');
+
+  for (const entry of stringSplit) {
+    if (entry.includes(':')) {
+      const index = entry.indexOf(':');
+      const name = entry.slice(0, index).trim();
+      const value = entry.slice(index + 1, entry.length).trim();
```
Comment (DarkMistyRoom) [inline src/convertToObject.js:12]: `const splitedEntry = entry.split(':');
const name = splitedEntry[0].trim();
...`

Comment (DarkMistyRoom) [reply src/convertToObject.js:12]: It looks simpler, isn't it?)

File: src/convertToObject.js [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const sourceStringTrimmed = sourceString.trim();
```
Comment (DarkMistyRoom) [inline src/convertToObject.js:5]: no need to trim string here because these spaces will be trimmed in 11-12 lines

**Review by DarkMistyRoom — APPROVED** [review]

**Pull request closed without merging**